community-operators/jaeger/1.16.0/jaeger.v1.16.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,373 @@
gOTMuNzUgMzU1LjY1IDkzLjA1IDM1Ni4wNQpMIDkzLjIgMzU2LjA1ClEgOTMuMyAzNTYuMDUgOTMuMzUgMzU2LjA1IDk1LjA1IDM1NS45NSA5Ni44NSAzNTUuOTUgOTYuODUgMzU1Ljg1IDk2LjkgMzU1Ljg1IDk4LjYgMzU1LjM1IDk5LjQ1IDM1NCA5OS40NSAzNTMuOSA5OS40NSAzNTMuODUgOTkuOTUgMzUyLjM1IDk5LjIgMzUxLjIgOTguNyAzNTAuNzUgOTguMTUgMzUwLjQgOTcuOTUgMzUwLjI1IDk3LjggMzUwLjIgOTIuMzUgMzQ5IDg1LjkgMzQ5IDg1Ljg1IDM0OSA4NS44IDM0OSA4My44NSAzNDguODUgODEuOCAzNDguODUgODEuOCAzNDguOCA4MS43NSAzNDguOCA4MC45IDM0OC40NSA4MC4zIDM0Ny44CkwgODAuMyAzNDcuNwpRIDc5Ljg1IDM0Ny4xNSA3OS44NSAzNDYuMiA4MCAzNDUuOSA4MC4yIDM0NS42IDgwLjIgMzQ1LjU1IDgwLjIgMzQ1LjQ1IDgwLjcgMzQ1LjA1IDgxLjE1IDM0NC42IDgxLjE1IDM0NC41NSA4MS4xNSAzNDQuNSA4MS43IDM0NC4yNSA4Mi4zIDM0NC4wNSA4Mi4zNSAzNDQgODIuNCAzNDQgODMuMyAzNDQuMiA4NC4yIDM0My45NSA4NC42NSAzNDMuOCA4NS4xIDM0My41NSA4NS4xNSAzNDMuNTUgODUuMiAzNDMuNTUKTCA4NS4yIDM0My41ClEgODYuNDUgMzQyLjc1IDg4LjEgMzQyLjQ1IDkyLjIgMzQxLjg1IDk1LjkgMzQwLjgKTCA5NS45IDM0MC43ClEgOTUuOSAzNDAuNjUgOTUuOSAzNDAuNiA5Ni43NSAzNDAuMyA5Ny43NSAzNDAuMSA5Ny44IDM0MC4xIDk3Ljg1IDM0MC4xIDk4LjE1IDMzOS45IDk4LjM1IDMzOS42NSA5OC40IDMzOS42NSA5OC40NSAzMzkuNjUgOTkgMzM5LjY1IDk5LjQ1IDMzOS41IDk5LjUgMzM5LjUgOTkuNTUgMzM5LjUgOTkuOTUgMzM5LjQgMTAwLjIgMzM5LjE1CkwgMTAwLjMgMzM5LjE1ClEgMTAyLjM1IDMzOC45IDEwNC40IDMzOC43NSAxMDYuMjUgMzM4LjY1IDEwNy44IDMzOC4zCkwgMTA3LjggMzM4LjI1ClEgMTA3Ljc1IDMzOC4yIDEwNy44IDMzOC4wNSAxMDkuMjUgMzM4LjYgMTExLjI1IDMzOC4yNSAxMjAuNjUgMzM2LjYgMTMwLjMgMzM1LjIgMTMxLjg1IDMzNSAxMzMuNSAzMzQuNzUKTCAxMzMuNSAzMzQuNwpRIDEzNS4yNSAzMzMuODUgMTM3Ljc1IDMzMy44IDEzOC4zIDMzMy4zIDEzOS4zNSAzMzMuMzUgMTM5LjY1IDMzMy40IDEzOS45IDMzMy4zNSAxNDIuNzUgMzMyLjkgMTQ1LjA1IDMzMS44NSAxNDUuMDUgMzMxLjggMTQ1LjEgMzMxLjc1IDE0NS40NSAzMzEuNDUgMTQ1LjU1IDMzMC44NSAxNDUuNTUgMzMwLjggMTQ1LjUgMzMwLjc1IDE0NC40IDMyOS40NSAxNDIuNSAzMjguOSAxMzcuNyAzMjguOCAxMzMgMzI4LjQ1IDEyOC41NSAzMjguMSAxMjQuMSAzMjcuNzUgMTE4Ljg1IDMyNy4zNSAxMTMuNTUgMzI3LjcgMTA3IDMyOC42IDEwMi4zNSAzMzEuMzUgOTguODUgMzMzLjM1IDk1LjMgMzM1LjI1IDk1LjI1IDMzNS4yNSA5NS4xNSAzMzUuMjUgOTIuNyAzMzUuNyA5MC42NSAzMzYuNgpMIDkwLjY1IDMzNi42NQpRIDkxLjk1IDMzNi4zIDkzLjIgMzM2LjQ1IDkzLjE1IDMzNi40NSA5My4xIDMzNi41IDg5LjUgMzM3LjcgODUuOCAzMzguOSA4Ni42NSAzMzcuOTUgODguMSAzMzcuNjUgODkuNSAzMzcuMzUgOTAuNTUgMzM2LjcgOTAuNSAzMzYuNyA5MC40IDMzNi43IDg1Ljk1IDMzNy41NSA4Mi42NSAzMzkuNSA4Mi41NSAzMzkuNSA4Mi41IDMzOS41IDgwLjEgMzM5LjYgNzguMzUgMzQwLjM1IDc4LjMgMzQwLjM1IDc4LjI1IDM0MC4zNSA3NyAzNDAuMzUgNzYuMyAzNDAuODUgNzQuMTUgMzQxLjIgNzIuMDUgMzQxLjU1IFoiLz4KCjxwYXRoIGZpbGw9IiMyOTI5MjkiIGZpbGwtb3BhY2l0eT0iMC41NTI5NDExNzY0NzA1ODgzIiBzdHJva2U9Im5vbmUiIGQ9IgpNIDYwLjIgMzQzLjg1ClEgNTIuMyAzNDUuNTUgNDMuNTUgMzQ2LjMKTCA0My41NSAzNDYuNApRIDM5LjcgMzQ3LjMgMzguMSAzNTAuMzUgMzguMDUgMzUxLjIgMzguNCAzNTEuNzUgMzguNCAzNTEuOCAzOC40NSAzNTEuOCAzOC40NSAzNTEuNzUgMzguNDUgMzUxLjY1IDM4LjM1IDM1MS4xIDM4LjM1IDM1MC40NSAzOS43NSAzNDcuNDUgNDMuNDUgMzQ2LjcgNDMuNSAzNDYuNyA0My41NSAzNDYuNyA0OC4xIDM0Ni4xIDUyLjcgMzQ1LjY1IDU3LjggMzQ1LjIgNjEuODUgMzQzLjkgNjYuNjUgMzQyLjM1IDcyLjA1IDM0MS41NSA3NC4xNSAzNDEuMiA3Ni4zIDM0MC44NSA3NyAzNDAuMzUgNzguMjUgMzQwLjM1IDc4LjMgMzQwLjM1IDc4LjM1IDM0MC4zNSA4MC4xIDMzOS42IDgyLjUgMzM5LjUgODIuNTUgMzM5LjUgODIuNjUgMzM5LjUgODU
```

Comment:
, please chage it to 1.15.0 as 1.15.1 doesn't exist

Reply:
Done!